packages/pieces/monday/src/lib/triggers/new-update-trigger.ts

```diff
@@ -0,0 +1,47 @@
+import { pollingHelper } from "@activepieces/pieces-common"
+import { TriggerStrategy, createTrigger } from '@activepieces/pieces-framework'
+
+import { mondayProps } from '../common/props'
+import { updatesPolling } from "./polling"
+
+export const mondayNewUpdatesTrigger = createTrigger({
+  name: `monday_new_updates`,
+  displayName: 'New Update',
```

Comment:
@ashrafsamhouri @kanarelo I am not sure if the user will understand what that trigger means.

I don't understand but i am not familiar with monday

Reply:
Its a Monday thing... https://developer.monday.com/api-reference/docs/updates
I don't know much about monday. 
But we can always iterate on the pieces when the user requests it or any complaint.

Reply:
Yes, it's a Monday thing and Monday users should understand it!